modes/eglot/evil-collection-eglot.el

```diff
@@ -36,6 +36,7 @@
   "Set up `evil' bindings for `eglot'."
   (evil-collection-define-key 'normal 'eglot-mode-map
     "gd" 'xref-find-definitions
+    "gD" 'xref-find-definitions-other-frame
```

Comment:
`g5`? Since other-frame commands usually are under C-x 5 prefix.

Reply:
That could work to. I prefer the ergonomics of pressing `gD`  than having to reach for `5` on my keyboard but I realize that this may be subjective/personal preference.

Reply:
@condy0919 WDYT

Reply:
I prefer to leave it in `g5` and make `gD` for the other-window version. Frames are rare to use when compared with windows.

Reply:
Sounds good to me. I'll send an update today.